dosomething-dev/main.tf

```diff
@@ -52,10 +52,6 @@ provider "null" {
   version = "~> 2.1"
 }
 
-locals {
-  papertrail_log_format = "%t '%r' status=%>s app=%%{X-Application-Name}o cache=\"%%{X-Cache}o\" country=%%{X-Fastly-Country-Code}o ip=\"%a\" user-agent=\"%%{User-Agent}i\" service=%%{time.elapsed.msec}Vms"
-}
```

Comment:
After inlining this into the backend module in #270 and the frontend module here, this is no longer used!